src/Moryx.AbstractionLayer.Products.Endpoints/Model/RecipeModel.cs

```diff
@@ -49,13 +49,6 @@ public class RecipeModel
         [DataMember]
         public Entry Properties { get; set; }
 
-        /// <summary>
-        /// The id of the currently referenced workplan
-        /// </summary>
-        [Obsolete("Use WorkplanModel instead")]
-        [DataMember]
-        public long WorkplanId { get; set; }
```

Comment:
Have you checked if the WorkplanId is used in UI? You have to update the generated API too

Reply:
https://github.com/PHOENIXCONTACT/MORYX-Framework/commit/890dc18bc81fe8a441875394f5df78d1bac0f42f#diff-ee0466273ff3b3b94e00e07f839cf737b7086c945a607371e3e2613a581e648dL16
https://github.com/PHOENIXCONTACT/MORYX-Framework/commit/890dc18bc81fe8a441875394f5df78d1bac0f42f#diff-ef214057aad84d70bf517c803a03ed1ef5b3ec7bb21f98b955a21a0ecf1c3570L96